molecule/scenario_resources/tests/test_default.rb

```diff
@@ -7,7 +7,7 @@
   its('group') { should eq 'pulp' }
 end
 
-['pulpcore-api','pulpcore-content','pulpcore-resource-manager', 'pulpcore-worker@1', 'pulpcore-worker@2'].each do |pservice|
```

Comment:
how do I conditionally remove 'pulpcore-resource-manager'?
I don't know inspec/ruby

Reply:
Why conditionally? For these tests, it should be ok to just assume the "resource-manager" has gone.